README.md

```diff
@@ -10,7 +10,6 @@ Fork this repo and extend the skeletons with your own application code. ([check
 The project has the following dependencies:
```

Comment:
The README still contains the word 'client' multiple times, including the title.

Reply:
resolved